### ISSUE TITLE
Kétszintű hitelesítés miatti hiba

### ISSUE THREAD

**Issue by wroadd** [gala132/nCoreCouchPotatoProvider#12]
Ha aktív a kétszintű hitelesítés(amit nem lehet kikapcsolni), akkor nem lehet keresni a hagyományos login metódussal. Lehet, hogy erre nincs egyszerű megoldás.

**Comment by gala132** [comment]
Erre valóban nincs egyszerű megoldás. Lehethogy jó lenne írni a staffnak az oldalon, hogy ők is tudjanak róla és akkor az ő részükről kell majd valamiféle megoldás erre a problémára. Mi nem tudunk ezzel mit tenni sajnos.
Ebből az okból ezt lezárom és amint lesz valamilyen fejlemény ezzel kapcsolatban ami miatt nekünk kéne módosítani valamit a kódban itt, akkor újra meg lehet nyitni.

**Comment by gala132** [comment]
Ami még eszembejutott, az ideiglenes kódot meg lehetne adni egy harmadik mezőbe végülis a user+pass kombinációval. Mondjuk nem emlékszem, hogy a couchpotato hogyan működik pontosan, szóval kérdés, hogy elég-e lenne ha csökkentett biztonsági móddal lépnénk be és akkor elmentené a cookiekat? Feltéve ha képes rá.
@wroadd Te esetleg meg tudod próbálni megnézni?
Nem tudom mikor tudom majd megnézni jómagam, szóval ha valaki másnak lenne ideje, utánnanézhetne.
A trükk annyi lenne, hogy ha elmenti a cookiekat és mi csökkentett biztonság módban lépünk be majd, akkor az ideiglenes kód abban a rövid időben míg bejelentkezik jó lesz majd és akkor amíg a cookie érvényes akkor addig nem kell újra megadni az ideiglenes kódot és akkor így a CP-nek se kell bejelentkeznie majd újból. (Ez az elmélet)

**Comment by wroadd** [comment]
Igen, nekem is hasonló volt az elképzelésem, ezt próbáltam én is hegeszteni, csak idő hiányában nem jutottam messzire, de elvileg lehetséges a cookie-k továbbküldésével a CP-n belüli belépés
